### DISCUSSION TITLE
Using `_process` suffix to allow `date()` formatting in template results in wrong date

### DISCUSSION THREAD

**Discussion by vijay-prema** [kimai/kimai#5144] [Questions & Answers] [answered]
### Describe the issue

I create a template and use `{{ invoice['invoice.due_date_process']|date('d. m. Y.') }}` exactly as suggested in the [docs](https://www.kimai.org/documentation/invoices.html#template-variables) to format the date. Format works BUT the result in the generated PDF is just today's date, not the actual `invoice.due_date` which is 15 days from now.  Seems like the same problem happens for all the other date fields I tried such as `query.begin_process`.

When I use it without formatting (e.g. `{{ invoice['invoice.due_date'] }}`) then I always get the correct `invoice.due_date` showing.

### I already tried

- [X] I've read and searched [the documentation](https://www.kimai.org/documentation/).
- [X] I've searched for similar issues in this repository.
- [X] I've searched for similar issues in [the discussions](https://github.com/kimai/kimai/discussions).

### Kimai version

2.11.0

### How do you run Kimai?

Docker

### Which PHP version are you using?

Unknown

**Comment by kevinpapst** [comment, marked as answer]
Really 2.11 ? If so, that's the explanation. 
This old version doesn't have that variable, which means it uses null, which results in today.

Use the debug template from the docs to see which vars are available.
Or simply update your Kimai...

**Comment by vijay-prema** [comment]
Updated to 2.19.1 its working now, thank you
